### DISCUSSION TITLE
Add sprue instead of tab?

### DISCUSSION THREAD

**Discussion by kiwipaulrob** [5axes/SpoonAntiWarping#2] [Ideas]
https://youtu.be/MCcFMDv_4eo

**Comment by 5axes** [comment]
Need to be tested

**Comment by 5axes** [comment]
It's not a Sprue .. It's A Spoon
![image](https://user-images.githubusercontent.com/11015345/214027592-7505b011-3c5d-413f-9de5-e551d4610272.png)

[https://github.com/5axes/SpoonAntiWarping](https://github.com/5axes/SpoonAntiWarping)

**Comment by 5axes** [comment]
https://www.youtube.com/watch?v=K2niowptW7M
